src/main/kotlin/no/nb/bikube/catalogue/collections/model/dto/CollectionsUpdateDto.kt

```diff
@@ -0,0 +1,58 @@
+package no.nb.bikube.catalogue.collections.model.dto
+
+import kotlinx.serialization.*
+import no.nb.bikube.core.util.DateUtils
+import java.time.LocalDate
+import java.time.LocalTime
+
+// ExperimentalSerializationApi and @EncodeDefault are used to avoid serializing null values.
+// When they are not present, they will not be updated in Collections.
+// If they were null, the value in Collections would be 'null'.
+
+@Serializable
+class CollectionsUpdateDto @OptIn(ExperimentalSerializationApi::class) constructor(
```

Comment:
Kanskje litt uklart navn? Er dette bare en generell DTO for oppdatering av en post i Collections? Kanskje det burde hete noe sånt som CollectionsObjectUpdateDto eller CollectionsRecordUpdateDto 🤔

Reply:
Den er generell ja (non-optionals bør være der på alle typer). Tenkte først å bruke den på items også men trengte ikke det med infoen vi har, siden det meste skal ligge på manifestasjonen. Endrer navnet!